html_export/static/style.css

```diff
@@ -181,9 +181,9 @@ body {
 }
 
 @media (orientation: portrait), (max-aspect-ratio: 4/3), (max-width: 750px) {
-  .mobile-only {
+  /* .mobile-only {
     display: flex;
-  }
+  } */
```

Comment:
That shouldn't be just commented out. If the class is going away from the css, then you should remove the lines.

Reply:
Oh.


> Thanks for the contributions!

My pleasure